### PR TITLE
Added states

### PR DESCRIPTION
**Pull request by shabirjan**

### CODE REVIEW

File: GithubUsers-Redux/States/AppState.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//  GithubUsers-Redux
+//
+//  Created by Shabir Jan on 3/24/18.
+//  Copyright © 2018 Shabir Jan. All rights reserved.
+//
+
+import Foundation
+
+struct AppState {
+     let userList: UserListState
```
Comment (MaximBazarov) [inline GithubUsers-Redux/States/AppState.swift]: 5 spaces (ctrl+i)

File: GithubUsers-Redux/States/UserDetailState.swift [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+//  GithubUsers-Redux
+//
+//  Created by Shabir Jan on 3/24/18.
+//  Copyright © 2018 Shabir Jan. All rights reserved.
+//
+
+import Foundation
+
+struct UserDetailState {
+
```
Comment (MaximBazarov) [inline GithubUsers-Redux/States/UserDetailState.swift]: where is the state?

File: GithubUsers-Redux/States/UserListState.swift [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+//  GithubUsers-Redux
+//
+//  Created by Shabir Jan on 3/24/18.
+//  Copyright © 2018 Shabir Jan. All rights reserved.
+//
+
+import Foundation
+
+struct UserListState {
+    let users: [User]
```
Comment (MaximBazarov) [inline GithubUsers-Redux/States/UserListState.swift:12]: that means that it's only one possible state and this is when you have an array of users, but there is also a state when you don't like `loading`

**Review by MaximBazarov — CHANGES_REQUESTED** [review]

File: GithubUsers-Redux/States/UserDetailState.swift (hunk 2 above)
Comment (shabirjan) [reply GithubUsers-Redux/States/UserDetailState.swift]: i wanted to create intial state first.

File: GithubUsers-Redux/States/UserListState.swift (hunk 3 above)
Comment (shabirjan) [reply GithubUsers-Redux/States/UserListState.swift:12]: what would that be?

File: GithubUsers-Redux/States/UserDetailState.swift (hunk 2 above)
Comment (shabirjan) [reply GithubUsers-Redux/States/UserDetailState.swift]: Wanted to create inital state first

**Pull request merged**